tests/foreman/cli/test_registration.py

```diff
@@ -16,8 +20,53 @@
 
 from robottelo.config import settings
 
+pytestmark = pytest.mark.tier1
+
+
+@pytest.mark.e2e
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_match('[^6].*')
+def test_host_registration_end_to_end(
+    module_org,
+    module_location,
+    module_ak_with_synced_repo,
+    module_target_sat,
+    module_capsule_configured,
+    rhel_contenthost,
+):
+    """Verify content host registration with global registration
+
+    :id: b6cd60ba-8069-11ed-ac61-83315855d126
+
+    :steps:
+        1. Register host with global registration template to Satellite and Capsule
+
+    :expectedresults: Host registered successfully
+    """
+    result = rhel_contenthost.register(
+        module_org, module_location, module_ak_with_synced_repo.name, satellite=module_target_sat
+    )
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+
+    # Update module_capsule_configured to include module_org/module_location
+    module_target_sat.cli.Capsule.update(
+        {
+            'name': module_capsule_configured.hostname,
+            'organization-ids': module_org.id,
+            'location-ids': module_location.id,
+        }
+    )
+    result = rhel_contenthost.register(
+        module_org,
+        module_location,
+        module_ak_with_synced_repo.name,
+        target=module_capsule_configured,
+        satellite=module_target_sat,
+        force=True,
+    )
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
```

Comment:
What about performing some additional checks? For instance, we can check that the host is really registered to the capsule, not to the satellite by checking the config value in `/etc/rhsm/rhsm.conf`.

Reply:
May be you can use a different org/lce when dealing with capsule and go by:
```
result = client.execute('subscription-manager identity')
    assert result.status == 0
    assert module_lce_library.name in result.stdout
    assert module_org.name in result.stdout
```

Reply:
Added checks for hostname and port, thanks for suggestion :)

Reply:
Great!
Warning: The following suggestions smell nitpicky. :smile: 
Parsing the config seems more right to me than `grep`-ing it.
![image](https://user-images.githubusercontent.com/20440883/212161758-f8714061-6601-4c05-b80a-97c1df09068c.png)

I can imagine this being a helper-ish function. As I hinted, this is most likely a subject to a different PR.

Reply:
Nice suggestion, updated in same PR